drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c

```diff
@@ -3281,6 +3282,7 @@ static int vchiq_probe(struct platform_device *pdev)
 	bcm2835_codec = vchiq_register_child(pdev, "bcm2835-codec");
 	bcm2835_camera = vchiq_register_child(pdev, "bcm2835-camera");
 	bcm2835_audio = vchiq_register_child(pdev, "bcm2835_audio");
+	bcm2835_isp = vchiq_register_child(pdev, "bcm2835-isp");
```

Comment:
I object to the whole principle behind this mechanism, but that's not your problem.